src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+
+namespace Iot.Device.Mcp23xxx
+{
+    public class Mcp23xxx : IDisposable
+    {
+        private GpioController _controller;
```

Comment:
I know you mentioned that this is not yet complete, but I don't see where are you using the _controller, or even where are you initializing it. I suppose you would probably want to initialize it in the constructor and then use it for the reads and writes by lowering the chip select pin to 0?

Reply:
I left in by mistake when removing the GPIO-related code.  It will eventually be used for big-bang supported code.

Reply:
Can we remove it for now given that it is not used? That way the history of when this guy is added will match when it was actually used for future reference. Not super important, but I think it might be useful.